### PR TITLE
FE: remove legacy pulse components

### PR DESCRIPTION
**Pull request by qnkhuat**
Targeting #35841, not master since we'll wait until we cut a new release branch for 48 to land this epic
Epic: https://github.com/metabase/metabase/issues/34147

### CODE REVIEW

File: e2e/test/scenarios/sharing/reproductions.cy.spec.js [hunk 1]
```diff
@@ -259,49 +259,6 @@ function setUpAlert(questionId) {
   });
 }
 
-describe("issue 16918", () => {
```
Comment (qnkhuat) [inline e2e/test/scenarios/sharing/reproductions.cy.spec.js:262]: i wonder if we need to replace this with a reproduction for dashboard subscriptions

Comment (nemanjaglumac) [reply e2e/test/scenarios/sharing/reproductions.cy.spec.js:262]: Absolutely yes!

Comment (qnkhuat) [reply e2e/test/scenarios/sharing/reproductions.cy.spec.js:262]: om, so could you help me with that? :D

Comment (nemanjaglumac) [reply e2e/test/scenarios/sharing/reproductions.cy.spec.js:262]: https://github.com/metabase/metabase/pull/35843/commits/43eaf41b45a5fcc683d0d2967e3f15b7fc913592

Comment (qnkhuat) [reply e2e/test/scenarios/sharing/reproductions.cy.spec.js:262]: thank you thank you!

**Comment by nemanjaglumac** [comment]
Removed myself from the reviewers because I contributed to the PR.

**Review by ranquild — APPROVED** [review]

**Pull request merged**